CloudOnMobile/Modules/Files/FilesDownloader.swift

```diff
@@ -11,27 +11,9 @@ struct MockedFilesDownloader: FilesDownloader {
     /// - TODO: Connect downloading files to FilesDownloader.
 
     func getFilesList() async -> [File] {
-        [
-            File(
-                name: "mexico.jpg",
-                size: Size(),
-                type: .image
-            ),
-            File(
-                name: "video.mp4",
-                size: Size(),
-                type: .video
-            ),
-            File(
-                name: "music.mp3",
-                size: Size(),
-                type: .music
-            ),
-            File(
-                name: "file.pdf",
-                size: Size(),
-                type: .other
-            )
-        ]
+        guard let data = listFiles() else { return [] }
```

Comment:
We should not use functions that are available everywhere, since it brakes SOLID principles.
We should create an instance for CoreWrapper and be able to inject it to downloader.
In this way we will be able to test it and mock data.
Add create an instance of CoreWrapper inside MainDependencyContainer, it should be a protocol to change it in Unit tests.
It will increase the compilation time, since it is available in all places.

Reply:
Fixed. I moved `listFiles()` back to `CommandManager` because we already have a working logic for it. In future commits `CoreWrapper ` and `CommandManager` will become one thing.

> It will increase the compilation time, since it is available in all places
How comes that a global symbol increase compilation time? It's just an entry in the symbol table, isn't it?